src/test/moves/multi_target.test.ts

```diff
@@ -21,160 +21,113 @@ describe("Moves - Multi target", () => {
+    const player2 = game.scene.getParty()[1];
+    const enemy1 = game.scene.getEnemyParty()[0];
+    const enemy2 = game.scene.getEnemyParty()[1];
 
-  it("1v2 - target all near enemies - damage decrase", () => checkDamageDecrease(game, Moves.HYPER_VOICE, true, false, true), TIMEOUT);
-
-  it("1v1 - target all near enemies - check modifier", () => checkTargetMultiplier(game, Moves.HYPER_VOICE, true, true, false), TIMEOUT);
-
-  it("2v2 (immune) - target all near enemies - check modifier", () => checkTargetMultiplier(game, Moves.HYPER_VOICE, false, false, true, Abilities.SOUNDPROOF), TIMEOUT);
-
-  it("2v2 (immune) - target all near enemies - damage decrase", () => checkDamageDecrease(game, Moves.HYPER_VOICE, false, false, true, Abilities.SOUNDPROOF), TIMEOUT);
+    game.move.select(Moves.EARTHQUAKE);
+    game.move.select(Moves.SPLASH, 1);
+    await game.setTurnOrder([BattlerIndex.PLAYER, BattlerIndex.PLAYER_2, BattlerIndex.ENEMY, BattlerIndex.ENEMY_2]);
 
-});
+    await game.phaseInterceptor.to("MoveEndPhase");
 
-async function checkTargetMultiplier(game: GameManager, attackMove: Moves, killAlly: boolean, killSecondEnemy: boolean, shouldMultiplied: boolean, oppAbility?: Abilities) {
-  // play an attack and check target count
-  game.override.enemyAbility(oppAbility ? oppAbility : Abilities.BALL_FETCH);
-  await game.startBattle();
+    const damagePlayer2Turn1 = player2.getMaxHp() - player2.hp;
+    const damageEnemy1Turn1 = enemy1.getMaxHp() - enemy1.hp;
 
-  const playerPokemonRepr = game.scene.getPlayerField();
+    player2.hp = player2.getMaxHp();
+    enemy1.hp = enemy1.getMaxHp();
 
-  killAllyAndEnemy(game, killAlly, killSecondEnemy);
+    await game.killPokemon(enemy2);
+    await game.toNextTurn();
 
-  const targetCount = getMoveTargets(playerPokemonRepr[0], attackMove).targets.length;
-  const targetMultiplier = targetCount > 1 ? 0.75 : 1;
+    game.move.select(Moves.EARTHQUAKE);
+    game.move.select(Moves.SPLASH, 1);
+    await game.setTurnOrder([BattlerIndex.PLAYER, BattlerIndex.PLAYER_2, BattlerIndex.ENEMY]);
 
-  if (shouldMultiplied) {
-    expect(targetMultiplier).toBe(0.75);
-  } else {
-    expect(targetMultiplier).toBe(1);
-  }
-}
+    await game.phaseInterceptor.to("MoveEndPhase");
 
-async function checkDamageDecrease(game: GameManager, attackMove: Moves, killAlly: boolean, killSecondEnemy: boolean, shouldDecreased: boolean, ability?: Abilities) {
-  // Tested combination on first turn, 1v1 on second turn
-  await game.classicMode.runToSummon([Species.EEVEE, Species.EEVEE]);
+    const damagePlayer2Turn2 = player2.getMaxHp() - player2.hp;
+    const damageEnemy1Turn2 = enemy1.getMaxHp() - enemy1.hp;
 
-  if (ability !== undefined) {
-    game.scene.getPlayerField()[1].abilityIndex = ability;
-    game.scene.getEnemyField()[1].abilityIndex = ability;
-  }
+    enemy1.hp = enemy1.getMaxHp();
 
-  game.move.select(Moves.SPLASH);
-  game.move.select(Moves.SPLASH, 1);
+    // Turn 1 = 3 targets, turn 2 = 2 targets
```

Comment:
nitting this because I got confused reading this initially
```suggestion
    // Turn 1: 3 targets, turn 2: 2 targets
```